lib/gherkin_lint.rb

```diff
@@ -31,69 +33,52 @@
 require 'gherkin_lint/linter/use_outline'
 require 'multi_json'
 require 'set'
-
+require 'gherkin_lint/configuration'
```

Comment:
Really, I'm not certain I see the value of the cli if the config file exists.  In fact, many of the linters you have could be configured.  You just decided on a hardcoded threshhold, but ideally it would be up to the individual user to decide.  I'd be more inclined to move in this direction.  Frankly, I don't see the configuration changing that significantly between runs, which leads me to think that the CLI approach is less necessary with this change.

It's up to you to decided this, but I think the CLI would get very complicated